src/main/webapp/index.html

```diff
@@ -14,9 +15,9 @@
    const WEBSITE_URL = "https://8080-74f086d4-db1e-49b6-b00a-2fb76ef31083.us-east1.cloudshell.dev/"
```

Comment:
If this is used multiple times, might be a better idea to move it to a constants file